### PR TITLE
feat: tooltip component

### PR DESCRIPTION
**Pull request by boyarskiy**
https://autoricardo.atlassian.net/browse/VSST-414

## Motivation and context

Tooltip component

## Before

No tooltip

## After

Screenshot 2022-10-17 at 22 24 37

## How to test

https://boyarskiy-tooltip.components-pkg.branch.autoscout24.ch/?path=/story/components-utils-tooltip--overview

### CODE REVIEW

File: src/components/tooltip/index.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React, { FC } from 'react';
+
+import { Tooltip as ChakraTooltip, TooltipProps } from '@chakra-ui/react';
+
+const Tooltip: FC<TooltipProps> = ({ children, ...props }) => {
+  return (
+    <ChakraTooltip bg="gray.900" color="white" p="sm" hasArrow {...props}>
```
Comment (GutierrezGo) [inline src/components/tooltip/index.tsx:7]: Looking good!

One thing about the styles. What we do with the stylings is adding them into the themes.
Please move these styles in the corresponding component theme. 

Thank you!

File: src/components/tooltip/index.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs';
+
+import Tooltip from './index.tsx';
+
+<Meta title="Components/Utils/Tooltip" component={Tooltip} />
```
Comment (GutierrezGo) [inline src/components/tooltip/index.stories.mdx:5]: I think here should be in the subcategory `Overlay` instead `Utils`. 

```suggestion
<Meta title="Components/Overlay/Tooltip" component={Tooltip} />
```

Please adjust is. Thanks!

**Review by GutierrezGo — COMMENTED** [review]
One question regarding the icon.
Shouldn't we add the info tooltip icon for its future use?

File: src/components/tooltip/index.tsx (hunk 1 above)
Comment (GutierrezGo) [reply src/components/tooltip/index.tsx]: I think you can follow this theme to make the background work -> https://github.com/chakra-ui/chakra-ui/blob/main/packages/components/theme/src/components/tooltip.ts

Comment (boyarskiy) [reply src/components/tooltip/index.tsx]: @GutierrezGo thanks for the tip, updated

File: src/components/tooltip/__tests__/Tooltip.Test.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    </Tooltip>
+  );
+
+describe('<Tooltip />', () => {
+  it('does not show tooltip label', async () => {
+    renderWrapper();
+    expect(screen.queryByText(tooltipLabel)).not.toBeInTheDocument();
+  });
+
+  it('does show tooltip label on focus in', async () => {
```
Comment (GutierrezGo) [inline src/components/tooltip/__tests__/Tooltip.Test.tsx:22]:
```suggestion
  it('shows tooltip label on focus in', async () => {
```

File: src/components/tooltip/__tests__/Tooltip.Test.tsx [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    renderWrapper();
+    fireEvent.focusIn(screen.getByTestId('test-tooltip'));
+    expect(await screen.findByText(tooltipLabel)).toBeInTheDocument();
+  });
+
+  it('does not show tooltip label on focus out', async () => {
+    renderWrapper();
+    fireEvent.focusOut(screen.getByTestId('test-tooltip'));
+    expect(screen.queryByText(tooltipLabel)).not.toBeInTheDocument();
+  });
```
Comment (GutierrezGo) [inline src/components/tooltip/__tests__/Tooltip.Test.tsx:32]: I'm wondering why you need async without the promise. Do you really need the async?

**Pull request merged**